tensorflow/python/eager/pywrap_gradient_exclusions.cc

```diff
@@ -745,7 +745,7 @@ bool OpGradientDoesntRequireOutputIndices(
           {"SparseSegmentSum", {true, {}}},
           {"SparseSegmentSumWithNumSegments", {true, {}}},
           {"SparseSlice", {false, {1, 2}}},
-          {"SparseSoftmaxCrossEntropyWithLogits", {false, {0}}},
+          {"SparseSoftmaxCrossEntropyWithLogits", {false, {}}},
```

Comment:
Isn't this only using the second output still? Not sure this needs to change

It might be cleaner if you can just re-run the update script: https://github.com/tensorflow/tensorflow/blob/392249966a19d0f1923dc1e1b743ccdfb2887ee6/tensorflow/python/eager/gradient_input_output_exclusions.py#L19

If that runs into issues we can ping @saxenasaurabh for help. Sorry for the major merge conflict.

Reply:
Yep, I think you got the inputs and outputs switched. +1 to running the update script. There is a test that checks that `pywrap_gradient_exclusions.cc` matches the update script output so that would fail if there isn't an exact match. (Hopefully we will get rid of the circular dependencies and convert this into a genrule).

Thanks!

Reply:
@allenlavoie you are right, only the second output is required and I fixed by running the script as you proposed.